completions/rsync

```diff
@@ -59,7 +78,15 @@ _rsync()
                     break
                 fi
             done
-            [[ $shell == ssh ]] && _comp_xfunc ssh scp_remote_files
+            if [[ $shell == ssh ]]; then
+                local rsync_version=$(rsync --version | sed -n '1s/.*rsync *version \([0-9.]*\).*/\1/p')
+                _comp_cmd_rsync__vercomp "$rsync_version" "3.2.4"
+                if [[ $? == "2" ]]; then
```

Comment:
`$?` is numeric, better to use arithmetic evaluation.
```suggestion
                if (($? == 2)); then
```

Reply:
@Rogach Thank you for removing the double quotation, but the current version of PR still uses the conditional command `[[ ... ]]`. Could you switch from `[[ $? == 2 ]]` to the arithmetic command `(($? == 2))`?